### PR TITLE
icon link issue fixed

### LINKED ISSUE #268: Stand alone link and button styling
Links and button outside a block should be styled accordingly:

Link
**Primary button**
_Secondary button_

Should also apply to other blocks where no explicit button styling is set, e.g. cards, columns

### PR DESCRIPTION
**Pull request by sanmishr**
Fix #268 

 
- Before: https://main--hlx-test--urfuwo.hlx.live/topics/
- After: https://268-link-issue--hlx-test--urfuwo.hlx.page/topics/

### CODE REVIEW

**Review by karlpauls — CHANGES_REQUESTED** [review]
Does this really only belong into the promo? I thought this is more general. Furthermore, why do you need javascript for this?

**Comment by mhaack** [comment]
> Does this really only belong into the promo? I thought this is more general. Furthermore, why do you need javascript for this

+1 this should be only some classes in styles.css

File: aemedge/blocks/promo/promo.js [hunk 1]
```diff
@@ -1,5 +1,17 @@
-import { decorateButtons } from '../../scripts/aem.js';
+import { decorateButtons, decorateIcons } from '../../scripts/aem.js';
 
 export default async function decorate(block) {
   decorateButtons(block);
+
+  const parentElems = document.querySelectorAll('.default-content-wrapper .button-container');
```
Comment (rootpea) [inline aemedge/blocks/promo/promo.js:6]: this will not run if there is no promo block on a page

**Comment by sanmishr** [comment]
Closing this request as this is fixed .

**Pull request closed without merging**